### DISCUSSION TITLE
Can't get alf.io to properly work (docker on Synology NAS)

### DISCUSSION THREAD

**Discussion by spudje** [alfio-event/alf.io#1323] [Deployment]
Hello all,

Came across Alf.io triggered by this article https://mariushosting.com/how-to-install-alf-io-on-your-synology-nas/
I followed it, with one exception:

I don't use a wildcard synology (let's encrypt) certifcate, but use my own domain with own certificate. Currently for testing I deployed it publicly on port 555, the router does portforwarding to the NAS keeping it at 555. Reverse proxy on NAS is mapping 555 to docker 7189.

I can login when opening https://<mydomain:555>/admin
After that I get the admin page with only the menu bar. Clicking on any of the menu items does not do anthing.

In the docker log I see this warning: [qtp1293940401-18] WARN  alfio.manager.system.ConfigurationManager - cannot find a value for BASE_URL
But I have no clue where to set this.

Thanks for any help!

**Comment by tengu1234** [comment]
I'm also having the same issue. any suggestions?

**Comment by cbellone** [comment]
Hi all! It's great to see people using alf.io in ways we would never expect :)
We didn't know about this tutorial, thank you for sharing!

With that being said, the option you're looking for is on STEP 20 of the linked guide. 
The field is "Base Application Url".
However, the tutorial contains an error: Base application Url must be a valid URL, i.e. `https://alf-io.mariushosting.synology.me` instead of just `alf-io.mariushosting.synology.me`

I have zero experience with synology NAS, so I believe the best option for you would be to ask the guide author for help

**Comment by tengu1234** [comment]
> Hello all,
> 
> Came across Alf.io triggered by this article https://mariushosting.com/how-to-install-alf-io-on-your-synology-nas/ I followed it, with one exception:
> 
> I don't use a wildcard synology (let's encrypt) certifcate, but use my own domain with own certificate. Currently for testing I deployed it publicly on port 555, the router does portforwarding to the NAS keeping it at 555. Reverse proxy on NAS is mapping 555 to docker 7189.
> 
> I can login when opening https://mydomain:555/admin After that I get the admin page with only the menu bar. Clicking on any of the menu items does not do anthing.
> 
> In the docker log I see this warning: [qtp1293940401-18] WARN alfio.manager.system.ConfigurationManager - cannot find a value for BASE_URL But I have no clue where to set this.
> 
> Thanks for any help!

Hi there, 

Not sure if you are still having the issue but I was having the same thing happen and discovered that on my desktop no matter which browser I use it will not work. I have to log in my laptop in order for it to work. 

Can't explain why but it's what way it is. so I would suggest trying on a different machine if possible.

**Comment by tengu1234** [comment]
So this issue has popped up again. Only seeing admin screen with menu only.

![image](https://github.com/alfio-event/alf.io/assets/25310265/b0ae3431-a2fe-4454-a5e7-955051435b1c)

I am wondering if there is a way to access the "Basic Configuration" menu after it's been deployed? to see if the fix from cbellone works.

I have put a lot of work into modifing the wording and settings which I don't want to do again if possible. :) 

Thanks
